src/docs/sdk/sessions.mdx

```diff
@@ -239,26 +239,66 @@ sessions over time.
 Generally speaking there are two separate modes for health reporting that SDKs
 can use. One is very short lived sessions, the other are user attended sessions.
 
-**Short lived sessions** *(req-mode)*
+**Short lived sessions** *(server-mode / request-mode)*
 
-These are sessions that correspond to a single HTTP request or RPC call.
+These are sessions roughly correspond to a HTTP requests or RPC calls in a
+server setting.
 
-- high in volume
+- high in volume, typically one session for each request
+- the number of sessions is usually higher than the number of events
```

Comment:
I'm not 100% clear what you mean here. More sessions than update events? More sessions than error events?

Or, perhaps a better question, what's the big idea you're trying to get across?

Reply:
I mean the `capture_{message,event,exception}` events. Is there another term that makes it more clear?